eodag/rest/utils.py

```diff
@@ -686,7 +686,11 @@ def download_stac_item_by_id_stream(catalogs, item_id, provider=None):
                 "content-disposition": f"attachment; filename={os.path.basename(filepath_to_stream)}",
             },
         )
-
+    except NotAvailableError:
+        download_stream_dict = dict(
+            content=(i for i in range(0)),
+            status_code=202,
+        )
```

Comment:
```suggestion
    except NotAvailableError as e:
        return ORJSONResponse(
            status_code=202,
            content={"description": str(e)},
        )
```

Reply:
Added in the commit https://github.com/CS-SI/eodag/commit/a35b21c653ceb890b2f5139a4fc71f4f46511882 with updated tests.